### DISCUSSION TITLE
How do i set setTransformer in java?

### DISCUSSION THREAD

**Discussion by tomleejumah** [AppIntro/AppIntro#1168] [Q&A] [answered]
tried  setTransformer(AppIntroPageTransformerType.Depth); but the interface method cannot be overriden

seccond is how can I animate the views inside my custom layout

**Comment by cortinico** [comment, marked as answer]
There is an example in the sample app:
https://github.com/AppIntro/AppIntro/blob/300bcebaaa2fbab81d455901e8e74e6315c315a8/example/src/main/java/com/github/appintro/example/ui/java/JavaIntro.java#L46C1-L47C67

**Comment by tomleejumah** [comment]
@cortinico Thanks...realised I had extendended AppIntro for paolorotolo and not cortinico
